config/config.toml.example

```diff
@@ -37,6 +37,9 @@ format = "text"
 # Disable automatic timestamps in output
 disable-timestamp = false
 
+# Stores slow query log into seperate files.
+#slow-query-file = ""
```

Comment:
Remove `#`?

Reply:
We should not set it by default, merge the output of slow query to standard log will be helpful for debugging.